### DISCUSSION TITLE
Request for comprehensive model benchmarking and checkpoint accessibility.

### DISCUSSION THREAD

**Discussion by YiLongSun** [open-edge-platform/anomalib#2030] [General]
Would it be possible to compile benchmark performance data for all models into a comprehensive README file? Additionally, could you provide links to their respective checkpoints on a shared drive for easy access?

**Comment by samet-akcay** [comment]
You could follow this PR https://github.com/openvinotoolkit/anomalib/pull/2005. Once merged, we will create a table. Regarding the checkpoints, we could discuss that with them. @djdameln , @ashwinvaidya17 any thoughts?
